examples/210-embedding_models.ipynb

```diff
@@ -0,0 +1,1128 @@
+{
```

Comment:
<p>Have you tried to improve the result somehow? It looks bad.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/etna-team/etna/pull/304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
- I choose not so big and not so small dataset (it was the only dataset that fits the parameters)
- I tried to search for optimal training args (but results were almost the same +- 1 smape)

Reply:
<p>Have you tried to change <code>output_dims</code>?</p>

Reply:
Of course, from ~1 to ~30